src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -177,6 +181,7 @@ export class FacetValueSuggestions extends Component {
     }
 
     $$(this.root).on(OmniboxEvents.populateOmniboxSuggestions, (e: Event, args: IPopulateOmniboxSuggestionsEventArgs) => {
+      this.suggestionHTMLRepresentation = new Map();
```

Comment:
Since the type of suggestionHTMLRepresentation is Map<>, assigning it an object throws an error. However, I could change the type to `{ [keyword: string] : string }`.

As for the reset statement, I'll remove it from here and move it to `getQuerySuggestionsKeywords`.

Reply:
For the type, you could use `Record<string, string>`

Reply:
Fixed, I refactored keywords, removing the need for it.